Lombiq.Tests.UI/Services/SqlServerManager.cs

```diff
@@ -122,7 +120,7 @@ public SqlServerRunningContext CreateDatabase()
         var filePathLocal = GetSnapshotFilePath(snapshotDirectoryPathLocal ?? snapshotDirectoryPathRemote);
         var directoryPathLocal =
             Path.GetDirectoryName(filePathLocal) ??
-            throw new InvalidOperandException($"Failed to get the directory path for local path \"{filePathLocal}\".");
```

Comment:
What the heck was this? :D

Reply:
Some SQL related type mismatch exception. It keeps getting suggested when I type "invalidop", if the `Microsoft.SqlServer.Management.Dmf` namespace is already imported.